tests/Unit/Framework/TestHelpers.hpp

```diff
@@ -306,9 +309,14 @@ numerical_derivative(const Invocable& function,
   const std::array<double, VolumeDim> x_1behind = x - dx;
   const std::array<double, VolumeDim> x_2behind = x_1behind - dx;
   const std::array<double, VolumeDim> x_3behind = x_2behind - dx;
-  return (1.0 / (60.0 * delta)) * (function(x_3ahead) - function(x_3behind)) -
-         (3.0 / (20.0 * delta)) * (function(x_2ahead) - function(x_2behind)) +
-         (0.75 / delta) * (function(x_1ahead) - function(x_1behind));
+  return (1.0 / (60.0 * delta)) *
+             (function(x_3ahead, std::forward<Args>(args)...) -
+              function(x_3behind, std::forward<Args>(args)...)) -
```

Comment:
You shouldn't forward the same thing multiple times.  Forwarding implies that the object may moved from.

It's probably cleaner to wrap your function in a lambda, rather than modifying this helper.  For your uses you can just move your existing wrappers into the loops and capture the loop variables.

Reply:
good catch, I removed the commit.